wiki/Registration/en.md

```diff
@@ -19,3 +17,17 @@ Don't forget to read the [rules](/wiki/Rules) very carefully!
 Once that is done, you are now ready to start your rhythm adventure! You can [add beatmaps](/wiki/Installation#adding-beatmapsets) to play, [make a skin](/wiki/Skinning), or [make a beatmap](/wiki/Beatmapping). You could also stop by and say "Hi" in the [Introductions subforum](https://osu.ppy.sh/community/forums/8).
 
 If you need further in-game help, you can ask in the `#help` [Internet Relay Chat](/wiki/Internet_Relay_Chat) (IRC) channel by opening the chat console (press `F8` or `F9` in-game) then type `/join help`. You could also post your question in the [Help subforum](https://osu.ppy.sh/community/forums/5).
+
+## Troubleshooting
+
+**The servers could be having issues.**
+
+Check the [osu! server status page](https://status.ppy.sh/) to see if there are any ongoing issues with the servers.
+
+**This can also happen if you are using a shared internet connection, a proxy, or VPN.**
+
+Try using another proxy or VPN provider - if all else fails, send in a support ticket via mail to [accounts@ppy.sh](mailto:accounts@ppy.sh).
+
+If you are not using any of the above, then you will need to send in a support ticket, as your connection has likely been blacklisted in error.
+
+Tor or networks associated with the Tor relay will not connect to the game servers for the most part. This is to avoid malicious use.
```

Comment:
Now that you have removed the heading, I now realise that the contents doesn't make much sense. Sorry about that.

I don't think this is section is actually part of registration. It is just about trying to play online..? It should probably be under the gameplay section.

Reply:
moved the section under Gameplay in Help Centre, removed it from Registration